gluten-core/src/main/resources/substrait/proto/substrait/algebra.proto

```diff
@@ -268,6 +268,15 @@ message FetchRel {
   substrait.extensions.AdvancedExtension advanced_extension = 10;
 }
 
+// The relational operator representing TOP N calculation
+message TopNRel {
+  RelCommon common = 1;
+  Rel input = 2;
+  int64 n = 3;
+  repeated SortField sorts = 4;
+  substrait.extensions.AdvancedExtension advanced_extension = 10;
+}
```

Comment:
Could you document this change?
https://github.com/apache/incubator-gluten/blob/main/docs/developers/SubstraitModifications.md

Reply:
Could we utilize the SortRel operator to implement TopNRel, incorporating the 'n' parameter within the advanced_extension?

Reply:
Refer to this comment https://github.com/apache/incubator-gluten/pull/5409#discussion_r1566884037, let's make a new operator for better compatibility with other potential Substrait implementers.